extensions-api/src/main/scala/pl/touk/nussknacker/engine/api/process/ExpressionConfig.scala

```diff
@@ -45,7 +45,7 @@ object ExpressionConfig {
   val defaultStrictMethodsChecking            = true
   val defaultStaticMethodInvocationsChecking  = true
   val defaultMethodExecutionForUnknownAllowed = false
-  val defaultDynamicPropertyAccessAllowed     = false
+  val defaultDynamicPropertyAccessAllowed     = true
```

Comment:
We still want to have dynamic property access banned by defaut.
By dynamic property access I mean access to field that is not defined in record's definition, not indexing in general - take a look at comments below for details.

Reply:
dynamic property access is a concept that we added because we had a security leak - user could e.g. `#record['getClass']['getClassLoader'] etc.